examples/visualization/evoked_topomap.py

```diff
@@ -121,6 +121,24 @@
                     time_unit='s')
 plt.subplots_adjust(left=0.01, right=0.99, bottom=0.01, top=0.88)
 
+# %%
+# We can also highlight specific channels by adding a mask:
```

Comment:
I'd add a simpler example before this one. For example creating the mask by finding channel-time points that exceed some value. This way we first showcase the easier approach (and arguably more common) to mask construction.